cmd/build/helmify/static/templates/_helpers.tpl

```diff
@@ -1,3 +1,15 @@
+{{/*
+If createNamespace is set to true, sets the namespace to "gatekeeper-system"
```

Comment:
PR looks great! This is a little confusing, is it possible to use `createNamespace` to create any namespace?

`helm install ...` -> installs to gatekeeper-system
`helm install --namespace myNamespace ...` -> installs to myNamespace (creating namespace)
`helm install --namespace myNamespace --set createNamespace=false ...` -> installs to myNamespace (namespace should exist before)

Reply:
Hi @sozercan. I agree that the `createNamespace` is confusing and was implemented before this change. It is considered best practice to not create namespaces in a Helm chart. That being said here is how this works.

By default in the values.yaml file `createNamespace: true`

So with that

`helm install ....` -> Creates the `gatekeeper-system` namespace and resources are installed there. Helm will manage the chart in the default namespace.

`helm install --set createNamespace=false ...` -> no namespace is created and one isn't specified in the helm command so it is installed in the default namespace

`helm install -n myNamespace ...` -> gatekeeper-system namespace is created and helm installs the resources there, however the chart is managed in `myNamespace` namespace. This is what happens today as well.

`helm install -n myNamespace --set createNamespace=false ...` helm installs the resources to myNamespace and manages the chart there. 

When I say "manages the chart in this namespace", I am referring to if you where to run `helm list -n <namespace>` you would see charts "installed" in that namespace. The way it is now you are likely to install the resources in 1 namespace and have the chart managed in another.


Currently today how we install the chart is `helm install -n gatekeeper-system --set createNamespace=false gatekeeper gatekeeper/gatekeeper --create-namespace` This makes everything fairly clear what is going on and everything contained to 1 namespace, however we are stuck with using `gatekeeper-system` as our namespace.

Reply:
I didn't know there was a `--create-namespace` flag in Helm 3. I think we should deprecate the `createNamespace` in values when Helm 2 chart is removed.

Reply:
Do we need to update the readme/docs after we cut a new release to reflect these chart changes so we have better guidance for what is the recommended way to install this chart? 

ie. install all the gk resources and the chart releases in the same namespace vs install all the gk resources in the `gatekeeper-system` ns while the chart releases are in another namespace?